auth_oidc/views/auth_oauth_provider.xml

```diff
@@ -19,6 +19,16 @@
                 <field name="token_endpoint" />
                 <field name="jwks_uri" />
             </field>
+            <xpath expr="//sheet/group[last()]" position="after">
+                <group name="mappings">
+                    <field name="group_line_ids">
+                        <tree editable="bottom">
+                            <field name="group_id" />
+                            <field name="expression" />
```

Comment:
Some help / examples about what to fill-in in `expression` would be welcome.

Reply:
would

```suggestion
                            <field name="expression" placeholder="token['scope']['value'] == 'hello world'" />
```

suffice?

Reply:
Good enough for me :)

Reply:
and took this one